cockatrice/src/remoteclient.cpp

```diff
@@ -648,6 +672,7 @@ void RemoteClient::doSubmitForgotPasswordResetToServer(const QString &hostname,
     lastPort = port;
     token = _token.trimmed();
     password = _newpassword;
+    hashedPassword = "";
```

Comment:
if we're just gonna set it empty a bunch of times here, i like my idea of the default "" above

Reply:
this is shorter than QString() which I'd use otherwise